tests/handlers/test_appservice.py

```diff
@@ -124,7 +126,9 @@ def test_query_room_alias_exists(self):
             Mock(room_id=room_id, servers=servers)
         )
 
-        result = self.successResultOf(self.handler.query_room_alias_exists(room_alias))
+        result = self.successResultOf(
+            defer.ensureDeferred(self.handler.query_room_alias_exists(room_alias))
+        )
```

Comment:
Is there a difference between `self.successResultOf` and `self.get_success`?

Reply:
`successResultOf` doesn't handle async functions on old Twisted. Note that `get_success` is only available on `HomeserverTestCase`:

https://github.com/matrix-org/synapse/blob/4b73488e811714089ba447884dccb9b6ae3ac16c/tests/unittest.py#L480-L486